### ISSUE TITLE
myanimelist.net: ads

### ISSUE THREAD

**Issue by blindassassin111** [uBlockOrigin/uAssets#17018]
### Prerequisites

- [X] I read and understand the [policy about what is a valid filter issue](https://github.com/uBlockOrigin/uAssets/blob/master/README.md#uassets).
- [X] I verified that this issue is not a duplicate. (Use this [button](https://user-images.githubusercontent.com/585534/146582579-c32707a0-36a1-4cc5-ad50-83172c9f67a8.png) to find out.)
- [X] I forced an update of my filter lists. (Click the ["Purge all caches"](https://github.com/gorhill/uBlock/wiki/Dashboard:-Filter-lists#purge-all-caches) button while holding the 'Shift' key, then click the ["Update now"](https://github.com/gorhill/uBlock/wiki/Dashboard:-Filter-lists#update-now) button.)
- [X] I did not remove any of the [default filter lists](https://user-images.githubusercontent.com/585534/91296476-ab9d8c00-e76a-11ea-9fb9-87d6861bdf73.png), or I have verified that the issue was not caused by removing any of the default lists.
- [X] I did not enable external filter lists, or I have verified that the issue still occurs without enabling external filter lists.
- [X] I do not have custom filters/rules, or I have verified that the issue still occurs without custom filters/rules.
- [X] I am not using uBlock Origin (uBO) along with other [similarly-purposed blockers](https://twitter.com/gorhill/status/1033706103782170625).
- [X] I have verified that the web browser's built-in blocker or [DNS blocking](https://en.wikipedia.org/wiki/DNS_blocking) (standalone or through a VPN) is not causing the issue.
- [ ] I did not answer truthfully to **ALL** the above checkpoints.

### URL address of the web page

`https://myanimelist.net/`

### Category

ads

### Description

Banner and side ads are no longer blocked. (along with the accompanying background color change)
Tried solutions from previous issues for MAL but they didn't work.

### Screenshot(s)

<details>
<summary>Screenshot(s)</summary>

<img width="1077" alt="ads" src="https://user-images.githubusercontent.com/38090555/222648910-82b1a1a2-1905-44f3-85b0-e9acd8405e05.PNG">

</details>

### Configuration

<details>

```yaml
uBlock Origin: 1.47.2
Chromium: 110
filterset (summary): 
  network: 85144
  cosmetic: 39215
  scriptlet: 17355
  html: 0
listset (total-discarded, last-updated): 
  default: 
    user-filters: 21-0, never
    easylist: 63314-47, 9h.36m
    easyprivacy: 31003-1172, 9h.36m
    plowe-0: 3750-928, 9h.36m
    ublock-abuse: 78-0, 9h.36m
    ublock-badware: 4609-114, 9h.36m
    ublock-filters: 32756-68, 9h.36m
    ublock-privacy: 287-1, 9h.36m
    ublock-quick-fixes: 344-21, 9h.36m
    ublock-unbreak: 1921-58, 9h.36m
    urlhaus-1: 6216-0, 9h.36m
filterset (user): [array of 21 redacted]
trustedset: 
  added: [array of 4 redacted]
modifiedUserSettings: [none]
modifiedHiddenSettings: [none]
supportStats: 
  allReadyAfter: 2541 ms (selfie)
  maxAssetCacheWait: 130 ms
popupPanel: 
  blocked: 5
  blockedDetails: 
    myanimelist.net: 1
    google-analytics.com: 1
    googlesyndication.com: 1
    googletagmanager.com: 1
    googletagservices.com: 1
```
</details>

**Comment by mapx-** [comment]
Where do you get the ads ? home page, specific page ? links ? Provide a screenshot too.

**Comment by blindassassin111** [comment]
> Where do you get the ads ? home page, specific page ? links ? Provide a screenshot too.

Home page, And added the image to the original post.

**Comment by ghajini** [comment]
USA vpn 
@blindassassin111 
did you mean this

https://user-images.githubusercontent.com/20338483/222650444-28bb1e97-e0e8-4354-adfa-4a58daef1019.png
https://user-images.githubusercontent.com/20338483/222650469-76d77603-17bf-4a3a-a9ae-9e53280306bc.png

`myanimelist.net##div[data-koukoku-width]` works for me 

@ryanbr

**Comment by mapx-** [comment]
yes, that's the background color too to fix

@blindassassin111 
test

```
myanimelist.net##[data-koukoku-width]
myanimelist.net##body:style(background-color: white !important)
```

**Comment by blindassassin111** [comment]
> yes, that's the background color too to fix
> 
> @blindassassin111 test
> 
> ```
> myanimelist.net##[data-koukoku-width]
> myanimelist.net##body:style(background-color: white !important)
> ```

The first line works, the second one causes issues which was discussed in #12679

**Comment by mapx-** [comment]
Test only

`myanimelist.net##[data-koukoku-width]:remove()`

**Comment by dportvine** [comment]
Why not?
`myanimelist.net##body:style(background-color: initial !important)`
and add 
`myanimelist.net##body:style(background-color: #000000 !important)`
works for me

**Comment by blindassassin111** [comment]
> Test only
> 
> `myanimelist.net##[data-koukoku-width]:remove()`

This removed it all and didn't cause the issues from just changing the background color, Thank you.

**Comment by krystian3w** [comment]
> Why not? / and add / works for me

Overriding CSS backgrounds requires testing exotic layouts for this site when someone doesn't modify it globally in Stylus/Stylish (this one is automatically overwritten until the add-ons go to the [tabs.insertCSS()](https://developer.mozilla.org/en-US/docs/Mozilla/Add-ons/WebExtensions/API/tabs/insertCSS) aka [scripting.insertCSS()](https://developer.mozilla.org/en-US/docs/Mozilla/Add-ons/WebExtensions/API/scripting/insertCSS)).

Problems with exotic themes can be found in this repository:

> causes issues which was discussed in https://github.com/uBlockOrigin/uAssets/issues/12679https://github.com/uBlockOrigin/uAssets/issues/12854#issuecomment-1107439224 https://github.com/uBlockOrigin/uAssets/issues/12679#issuecomment-1095568556 https://github.com/uBlockOrigin/uAssets/issues/12679#issuecomment-1099408253 https://github.com/uBlockOrigin/uAssets/issues/12679#issuecomment-1107896216

**Comment by alltimemarr** [comment]
I'm having a different issue. I have uBlock Origin, but somehow certain elements are now blocked that have never been blocked before. For example, when visiting an anime page I can no longer see and click on a forum title, and when looking at the reviews, people's avatar image things are missing.

When I click the button in the extension to disable cosmetic filtering, those things re-appear, but so do ads.

Or when looking at someone's profile the buttons below their profile pic to do to the comment section, send a PM or gift membership are gone. Also the link to their reviews is missing. That all reappears with cosmetic filters turned off.

Example of this page with cosmetic filters turned on:
https://myanimelist.net/anime/35838/Shoujo_Shuumatsu_Ryokou

<img width="1646" height="556" alt="Image" src="https://github.com/user-attachments/assets/d826a58c-04b4-44d2-a794-bdca80bd6d95" />

<img width="1637" height="367" alt="Image" src="https://github.com/user-attachments/assets/d73f50df-2e34-4590-8980-15fb7cf5f32e" />

<img width="1618" height="1476" alt="Image" src="https://github.com/user-attachments/assets/45875855-929f-4244-86bd-d0003b000974" />

<img width="468" height="698" alt="Image" src="https://github.com/user-attachments/assets/2be53874-3987-4f0c-a486-2277044e0b93" />

Without cosmetic filters:

<img width="1635" height="543" alt="Image" src="https://github.com/user-attachments/assets/00d19806-1b3f-4548-a900-61ec324fe1a0" />

<img width="1648" height="358" alt="Image" src="https://github.com/user-attachments/assets/c0cd050b-bbc4-4512-a6dc-48382cca8614" />

<img width="1654" height="1561" alt="Image" src="https://github.com/user-attachments/assets/61e61260-80ee-4e2d-9d80-9e2e4cf392cf" />

<img width="475" height="1185" alt="Image" src="https://github.com/user-attachments/assets/a47cbba4-fd18-428a-9956-926976e46007" />

this is what I noticed right away on anime pages at least.

**Comment by alltimemarr** [comment]
I just did a something with inspect element, and I realized it has something to do with a class "ga-click". See these screenshots.
If I remove that line, everything comes back! but I don't know how to make it last.

<img width="1013" height="586" alt="Image" src="https://github.com/user-attachments/assets/35da0955-4849-4c25-9a0d-3adfc4c10a81" />

<img width="2318" height="1068" alt="Image" src="https://github.com/user-attachments/assets/c9c4e190-e58c-49ef-9bc8-d21743de42aa" />

----

Oh, I found even more!!

I think the quick fixes thing needs to be updated per the linked reddit post

! https://www.reddit.com/r/uBlockOrigin/comments/1sgy34h/ads_showing_on_myanimelist/
myanimelist.net##a.ga-click

myanimelist.net##:has(> .ga-click[rel])

the comment was made by a UBO team member, but it looks like the "wrong" line was used in the filter list in the end?
i have no clue how to override that, so I hope it can be fixed ^_^

**Comment by scallioncreamcheesebagel** [comment]
Came to confirm that uBlock Origin being enabled now breaks forum pages (example: https://myanimelist.net/forum/?topicid=2259111), no longer displaying usernames or user icons.

**Comment by Rexz4** [comment]
I'm also experimenting the same issues previous users stated. A lot of interactive elements are just completely blocked

**Comment by blindassassin111** [comment]
I am also having the same issue as mentioned above. It wasn't the case yesterday or the day before.

**Comment by stephenhawk8054** [comment]
Fixed here: https://github.com/uBlockOrigin/uAssets/commit/fc690ed83c5ca82c3eff34361cf15d00b091d678

**Comment by scallioncreamcheesebagel** [comment]
> Fixed here: [fc690ed](https://github.com/uBlockOrigin/uAssets/commit/fc690ed83c5ca82c3eff34361cf15d00b091d678)

Thank you!!

**Comment by gorhill** [comment]
Breakage (#33700) caused by `myanimelist.net##.js-scrollfix-bottom-rel.rightside > div[style]:has(.ad-sas)` in "uBlock filters -- Ads".

Related commit: https://github.com/uBlockOrigin/uAssets/commit/89dccdcd73c78d54e6c4ae4a4cda7e62c770e9be
